### ISSUE TITLE
PIT with nn.CrossEntropyLoss() from MIRNet

### ISSUE THREAD

**Issue by penguinwang96825** [asteroid-team/asteroid#596]
## 🚀 Feature
This is to assign speaker ID of each separated signals with PIT.

### Motivation

Authors from [MIRNet](http://www.interspeech2020.org/uploadfile/pdf/Thu-2-7-3.pdf) considers all of the loss terms by calculating every possible permutation of candidate pairs. They computed the PIT loss on estimated speaker identity information. In addition, they use cross-entropy loss with a classifier for the speaker embeddings.

### What you'd like

The entire training criterion is as follows: 
<img width="346" alt="截圖 2022-02-11 上午8 29 09" src="https://user-images.githubusercontent.com/28087825/153519757-502ced08-3573-472c-9449-c36509f6a8ac.png">

**Comment by mpariente** [comment]
This sounds very interesting, do you want to give it a try?
